rebar.config

```diff
@@ -63,9 +63,9 @@
 
 %% == Dependencies ==
 
-{deps, [ {elvis_core, "1.0.0"}
+{deps, [ {elvis_core, "1.1.1"}
```

Comment:
```suggestion
{deps, [ {elvis_core, "1.1.2"}
```

Reply:
Oh, goody. I'll have to update `rebar.lock` also, so the suggestion will be accepted via a push. Hope that's OK.